packages/web-components/src/components/va-accordion/va-accordion-item.e2e.ts

```diff
@@ -72,6 +105,22 @@ describe('va-accordion-item', () => {
     expect(button.getAttribute('aria-expanded')).toEqual('true');
   });
 
+  it('slot usage sets aria-expanded to true based on prop', async () => {
```

Comment:
We might not need this one since it doesn't look like it's related to the header changes.

Reply:
Removed with most recent commit